glib/src/lib.rs

```diff
@@ -156,13 +156,14 @@ mod utils;
 pub use self::utils::*;
 mod main_context;
 mod main_context_channel;
+pub use self::main_context::MainContextAcquireGuard;
+pub use self::main_context_channel::{Receiver, Sender, SyncSender};
+mod date;
```

Comment:
Before you ask, for some arbitrary reason `cargo fmt` wants to move `mod date` up here now :man_shrugging:

Reply:
Ok I won't ask then. :)